### ISSUE TITLE
Difference between service broker

### ISSUE THREAD

**Issue by rothgar** [aws/aws-controllers-k8s#19]
I'd like help understanding the goals of aws-service-operator compared to the existing [service broker](https://svc-cat.io/). Is it intended that both products will exist and overlap in functionality or will there be features/resources expected that will only be possible in one vs the other?

Looking at the existing/[old implementation for the service operator](https://operatorhub.io/operator/aws-service) I can see what AWS services are targeted but it would be nice to have clarity for customers on when they should use one vs another.

**Comment by ddseapy** [comment]
This was previously in the readme, which may help.

https://github.com/amazon-archives/aws-service-operator/tree/a804819fe16351bf10714cfb393a1934ef16d09c#faq

**Comment by mhausenblas** [comment]
TL;DR: for the time being, people using the service broker should continue to use it and we're coordinating with the maintainers to provide a unified solution.

Thanks for raising this question @rothgar and as pointed out by @ddseapy there's some info available already but let me expand on this. The service broker OSS project is also an AWS activity that, with the general shift of focus in the community from SB to operators, can be considered less actively developed. There are a number of things around ALM that the SB currently covers and which are at this juncture not yet covered by the scope of ASO, however we expect in the mid to long run that these two projects converge. We had AWS-internal discussions with the team that maintains the SB and we're on the same page concerning a unified solution.

We appreciate input and advice concerning features that are currently covered by SB only, for example bind/unbind or cataloging and looking forward to learn from the community how they are using SB so that we can take this into account.

**Comment by rothgar** [comment]
Great! Thanks for clarifying. That helps me understand the state and eventual scope.

**Comment by mhausenblas** [comment]
Re-opening this for visibility. We need to get that into an FAQ.

**Comment by mhausenblas** [comment]
Now in the FAQ: https://aws.github.io/aws-controllers-k8s/community/faq/#service-broker
